BPOR/BPOR.Rms/Controllers/ResearcherController.cs

```diff
@@ -468,7 +468,7 @@ public async Task<IActionResult> Edit(int id, int field)
     [HttpPost]
     [ValidateAntiForgeryToken]
     public async Task<IActionResult> Edit(int id,
-        [Bind("ShortName,ChiefInvestigator,StudySponsors,CPMSId,PortfolioSubmissionStatus,OutcomeOfSubmission," +
+        [Bind("ShortName,ChiefInvestigator,StudySponsors,CPMSId,PortfolioSubmissionStatus,OutcomeOfSubmission,Id," +
```

Comment:
If we need to `Bind` it is probably a sign the model is too complicated.
The model should only include properties that we want to bind anyway.

`Id` is already passed in as a route parameter so arguably we should have a few different models depending on what is needed.

Reply:
I went back over the method and noticed I overlooked a line near the end of the code
`model.Id = id;` 
this is not reached when going through a dependancy question as that returns much further up.
would it be better to remove the bound id and move this line nearer the start of the method, or is a restructure for the model still the better option?

Reply:
I think we could benefit from a slight larger restructuring of things in this area, but if we want to keep things compact to close out this fix I would be fine with moving `model.Id = id;` right up to the first line of the method.